File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/ordpres/d/PresPrintParamDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.ordpres.d
{
    /// <summary>
    /// PresPrintParamDTO 的摘要说明。
    /// </summary>
    public class PresPrintParamDTO : BaseDTO {

        public PresPrintParamDTO() {
 
        }

        /// <summary>
        /// 就诊
        /// </summary>
		public string Id_en {
            get { return getAttrVal<string>("Id_en",null); }
            set { setAttrVal<string>("Id_en", value); }
        }

        /// <summary>
        /// 处方串（逗号分隔）
        /// </summary>
		public string Id_preses {
            get { return getAttrVal<string>("Id_preses",null); }
            set { setAttrVal<string>("Id_preses", value); }
        }

        /// <summary>
        /// 年龄
        /// </summary>
		public double? Age {
            get { return getAttrVal<FDouble>("Age",null); }
            set { setAttrVal<FDouble>("Age", value); }
        }

        /// <summary>
        /// 过敏史
        /// </summary>
		public string Des_alcla {
            get { return getAttrVal<string>("Des_alcla",null); }
            set { setAttrVal<string>("Des_alcla", value); }
        }

        /// <summary>
        /// 客户扩展字段1
        /// </summary>
		public string Def1 {
            get { return getAttrVal<string>("Def1",null); }
            set { setAttrVal<string>("Def1", value); }
        }

        /// <summary>
        /// 客户扩展字段2
        /// </summary>
		public string Def2 {
            get { return getAttrVal<string>("Def2",null); }
            set { setAttrVal<string>("Def2", value); }
        }

        /// <summary>
        /// 客户扩展字段3
        /// </summary>
		public string Def3 {
            get { return getAttrVal<string>("Def3",null); }
            set { setAttrVal<string>("Def3", value); }
        }

        /// <summary>
        /// 客户扩展字段4
        /// </summary>
		public string Def4 {
            get { return getAttrVal<string>("Def4",null); }
            set { setAttrVal<string>("Def4", value); }
        }

        /// <summary>
        /// 客户扩展字段5
        /// </summary>
		public string Def5 {
            get { return getAttrVal<string>("Def5",null); }
            set { setAttrVal<string>("Def5", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_en", "Id_preses", "Age", "Des_alcla", "Def1", "Def2", "Def3", "Def4", "Def5"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.ordpres.d.PresPrintParamDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/ordpres/d/PresPrintParamDTO.java
package iih.ci.ord.dto.ordpres.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 医嘱处方打印查询入口参数 DTO数据 
 * 
 */
public class PresPrintParamDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 就诊
	 * @return String
	 */
	public String getId_en() {
		return ((String) getAttrVal("Id_en"));
	}
	/**
	 * 就诊
	 * @param Id_en
	 */
	public void setId_en(String Id_en) {
		setAttrVal("Id_en", Id_en);
	}
	/**
	 * 处方串（逗号分隔）
	 * @return String
	 */
	public String getId_preses() {
		return ((String) getAttrVal("Id_preses"));
	}
	/**
	 * 处方串（逗号分隔）
	 * @param Id_preses
	 */
	public void setId_preses(String Id_preses) {
		setAttrVal("Id_preses", Id_preses);
	}
	/**
	 * 年龄
	 * @return FDouble
	 */
	public FDouble getAge() {
		return ((FDouble) getAttrVal("Age"));
	}
	/**
	 * 年龄
	 * @param Age
	 */
	public void setAge(FDouble Age) {
		setAttrVal("Age", Age);
	}
	/**
	 * 过敏史
	 * @return String
	 */
	public String getDes_alcla() {
		return ((String) getAttrVal("Des_alcla"));
	}
	/**
	 * 过敏史
	 * @param Des_alcla
	 */
	public void setDes_alcla(String Des_alcla) {
		setAttrVal("Des_alcla", Des_alcla);
	}
	/**
	 * 客户扩展字段1
	 * @return String
	 */
	public String getDef1() {
		return ((String) getAttrVal("Def1"));
	}
	/**
	 * 客户扩展字段1
	 * @param Def1
	 */
	public void setDef1(String Def1) {
		setAttrVal("Def1", Def1);
	}
	/**
	 * 客户扩展字段2
	 * @return String
	 */
	public String getDef2() {
		return ((String) getAttrVal("Def2"));
	}
	/**
	 * 客户扩展字段2
	 * @param Def2
	 */
	public void setDef2(String Def2) {
		setAttrVal("Def2", Def2);
	}
	/**
	 * 客户扩展字段3
	 * @return String
	 */
	public String getDef3() {
		return ((String) getAttrVal("Def3"));
	}
	/**
	 * 客户扩展字段3
	 * @param Def3
	 */
	public void setDef3(String Def3) {
		setAttrVal("Def3", Def3);
	}
	/**
	 * 客户扩展字段4
	 * @return String
	 */
	public String getDef4() {
		return ((String) getAttrVal("Def4"));
	}
	/**
	 * 客户扩展字段4
	 * @param Def4
	 */
	public void setDef4(String Def4) {
		setAttrVal("Def4", Def4);
	}
	/**
	 * 客户扩展字段5
	 * @return String
	 */
	public String getDef5() {
		return ((String) getAttrVal("Def5"));
	}
	/**
	 * 客户扩展字段5
	 * @param Def5
	 */
	public void setDef5(String Def5) {
		setAttrVal("Def5", Def5);
	}
}